assert/assertions_test.go

```diff
@@ -2626,49 +2611,28 @@ func TestNoFileExists(t *testing.T) {
 	mockT = new(testing.T)
 	True(t, NoFileExists(mockT, "../_codegen"))
 
-	var tempFiles []string
-
-	link, err := getTempSymlinkPath("assertions.go")
-	if err != nil {
-		t.Fatal("could not create temp symlink, err:", err)
-	}
-	tempFiles = append(tempFiles, link)
+	link := getTempSymlinkPath(t, "assertions.go")
 	mockT = new(testing.T)
 	False(t, NoFileExists(mockT, link))
 
-	link, err = getTempSymlinkPath("non_existent_file")
-	if err != nil {
-		t.Fatal("could not create temp symlink, err:", err)
-	}
-	tempFiles = append(tempFiles, link)
+	link = getTempSymlinkPath(t, "non_existent_file")
 	mockT = new(testing.T)
 	False(t, NoFileExists(mockT, link))
-
-	errs := cleanUpTempFiles(tempFiles)
-	if len(errs) > 0 {
-		t.Fatal("could not clean up temporary files")
-	}
 }
 
-func getTempSymlinkPath(file string) (string, error) {
-	link := file + "_symlink"
-	err := os.Symlink(file, link)
-	return link, err
-}
+func getTempSymlinkPath(t *testing.T, file string) string {
```

Comment:
We can improve naming:
```suggestion
func tempSymlinkPath(t *testing.T, file string) string {
```